tests/integration/e2e/smpc_dp_operation_test.py

```diff
@@ -315,6 +315,54 @@ def test_gt(
     print("Published Result ", published_result.get())
 
 
+@pytest.mark.e2e
+def test_mean(
```

Comment:
Let's split this up and make two separate tests for PhiTensor mean and GammaTensor mean. The reason this is important is because publishing the result of PhiTensor.mean() is expected to take a LOT more privacy budget than GammaTensor.mean() (because there's only 1 data subject in PhiTensor- meaning their data has nowhere to hide). 
To accomodate for this, we could use a higher value of sigma for the PhiTensor test.

Lmk if any of this doesn't make sense- happy to talk it over :)

Reply:
I've split the DP mean method off into its own test, but lets review to make sure I'm setting it up as expected.